update all meta keys

dc-redeclipse opened the pull request:
update all meta keys lmeta. rmeta and start
(sorry Quin forget to update the others too)

dc-redeclipse committed:
update all meta keys

dc-redeclipse reviewed (commented):
That's pretty much the Windows 11 meta keys.
I am not sure if the icons will find the way back in the UI, but the resolution of this version is better and more modern and not directly classic windows on the first look.
The unified meta symbol I make matches the windows one more by accident, lol.

dc-redeclipse committed:
Merge branch 'redeclipse:master' into master

dc-redeclipse commented:
Not sure of this pr still needed, kinda lost track.

qreeves commented:
If I remember correctly, I expressed my concerns over the button not being recognizable enough.

dc-redeclipse committed:
Delete keys/rmeta.png

dc-redeclipse committed:
Delete keys/lmeta.png

dc-redeclipse committed:
Delete keys/start.png

dc-redeclipse committed:
classic meta style key

Replacing start.png, rmeta.png, and lmeta.png old Windows 7 keys with classic universal meta style keys symbol.
Maybe that's more noticeable.

IceflowRE commented:
Imho anyone who never saw a Mac will not know what they command key means.

qreeves commented:
Yeah, I think we should just leave this as-is for now. It'd be better to find an entirely new set of key icons.

dc-redeclipse commented:
I think these keys are no longer used anyway in the game cause the new UI using others. Maybe we can get rid of the completely.
